Examples/MagicWeather/MagicWeather.xcodeproj/project.pbxproj

```diff
@@ -389,7 +397,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 8SXR2327BM;
```

Comment:
these 2 lines won't work for folks outside our company, since it ties to our development team. Could you revert these? same for 412 / 420